ping/README.md

```diff
@@ -26,8 +26,12 @@ For Agent v7.21+ / v6.21+, follow the instructions below to install the ping che
    # Windows
    agent.exe integration install -t datadog-ping==<INTEGRATION_VERSION>
    ```
-
-2. Configure your integration similar to core [integrations][4].
+2. Install the `ping` binary in accordance to your OS. For example, run the following for Ubuntu:
+   ```shell
+   apt-get install iputils-ping
+   ```
+   
+3. Configure your integration similar to core [integrations][4].
```

Comment:
```suggestion
3. See the instructions in the [Integrations documentation][4]to configure your integration.
```

Reply:
Is there a specific section of that documentation that we can refer them to?

Reply:
I'm not completely sure, but I think we could also link to the [conf.yaml](https://github.com/DataDog/integrations-extras/blob/master/ping/datadog_checks/ping/data/conf.yaml.example) as an alternative?

Reply:
On second thought, step #3 might not be necessary, because the Configuration section tells user how to configure the conf.yaml file and links to it as well. https://a.cl.ly/v1uQQdWD What do you think?

Reply:
I think we should keep the current section since this is something the original maintainer added.